common/flatpak-transaction.c

```diff
@@ -695,6 +695,26 @@ flatpak_transaction_operation_get_related_to_op (FlatpakTransactionOperation *se
   return self->related_to_op;
 }
 
+/**
+ * flatpak_transaction_operation_get_is_skipped:
+ * @self: a #FlatpakTransactionOperation
+ *
+ * Gets whether this operation will be skipped when the transaction is run.
+ * Operations are skipped in some transaction situations, for example when an
+ * app has reached end of life and needs a rebase. By default, skipped
+ * operations are not returned by flatpak_transaction_get_operations() — but
+ * they can be accessed by traversing the operation graph using
+ * flatpak_transaction_operation_get_related_to_op().
```

Comment:
Would be nice to also add a note to the docs for `get_related_to_op()` to that effect.

Reply:
Done :+1: